tests/foreman/cli/test_auth.py

```diff
@@ -47,12 +47,12 @@ def configure_sessions(enable=True, add_default_creds=False):
             HAMMER_CONFIG, 'true' if enable else 'false'
         )
     )
```

Comment:
missed this refactor

Reply:
I meant https://github.com/SatelliteQE/robottelo/pull/8928/files#diff-0ad6a8b8ad0d00ed500f3f50d4c41d24171932e505be2f9a625acbddeefb8a28R44 was missed to convert